src/functions-and-arrays.js

```diff
@@ -52,14 +111,22 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let unique = arr.filter((value, index, array) => array.indexOf(value) === index);
+  if (arr.length === 0) return null;
+  return unique;
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr) {
```

Comment:
this function is supposed to receive an array and a word to search as arguments, right now it only receives an array.